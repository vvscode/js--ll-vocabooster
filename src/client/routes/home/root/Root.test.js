import React from 'react';
import { shallow } from 'enzyme';
import Component from './Root.component';

describe('Root', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {
      actions: {},
      route: {},
    };

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
