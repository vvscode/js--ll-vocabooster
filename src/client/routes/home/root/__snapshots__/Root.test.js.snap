// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root render should render with all props 1`] = `
<div
  className="root__container"
>
  <div
    className="root__wrapper"
  >
    <pre>
      {}
    </pre>
  </div>
</div>
`;
